filter-dialog.ts

```diff
@@ -151,11 +151,11 @@ export class FilterDialog extends LitElement {
           this.resetDialog();
         }}
       >
-        <div slot="headline">Edit Filter ${this.filterName}</div>
+        <div slot="headline">Edit Comparison Rules</div>
         <form slot="content" id="filterForm" method="dialog">
-          <md-filled-text-field
+          <md-outlined-text-field
             label="Filter name"
```

Comment:
```suggestion
            label="Name"
```

Should we call this `Name`, so as to avoid the term "Filter" altogether in the UI?

Reply:
agreed - I don't know how I even missed this one.